framework/auth/views.py

```diff
@@ -132,12 +138,16 @@ def forgot_password_get(auth):
     :param auth: the authentication context
     :return
     """
+    #overriding the routes.py sign in url to redirect to the dashboard after login
+    context = {}
+    next_url = request.url.replace('/forgotpassword/', '/dashboard/')
```

Comment:
I'd lean towards using `web_url_for`, which does a reverse lookup, and the `_absolute` parameter makes it an absolute url, instead of doing string replacement.
```
next_url = web_url_for('dashboard', _absolute=True)
```